ruby/ql/lib/codeql/ruby/ast/Literal.qll

```diff
@@ -41,6 +44,7 @@ class IntegerLiteral extends NumericLiteral instanceof IntegerLiteralImpl {
   /** Gets the numerical value of this integer literal. */
   final int getValue() { result = super.getValue() }
 
+  overlay[global]
```

Comment:
I would have thought that these were not needed when the root def has `overlay[global]`?